support-frontend/assets/helpers/contributions.ts

```diff
@@ -134,15 +134,35 @@ const getAmount = (
 	selectedAmounts: SelectedAmounts,
 	otherAmounts: OtherAmounts,
 	contributionType: ContributionType,
+	coverTransactionCostSelected?: boolean,
 ): number => {
 	const selectedAmount = selectedAmounts[contributionType];
 	const otherAmount = otherAmounts[contributionType].amount ?? '';
 
-	return parseFloat(
-		selectedAmount === 'other' ? otherAmount : selectedAmount.toString(),
+	// Only cover transaction costs for one off contributions
+	const coverTransactionCost =
+		coverTransactionCostSelected && contributionType === 'ONE_OFF';
+
+	return getAmountWithTransactionCostCovered(
+		parseFloat(
+			selectedAmount === 'other' ? otherAmount : selectedAmount.toString(),
+		),
+		coverTransactionCost,
 	);
 };
 
+function getAmountWithTransactionCostCovered(
+	amount: number,
+	coverTransactionCost?: boolean,
+): number {
+	if (coverTransactionCost) {
+		// Use the same rounding logic as the displayed price
+		return roundDp(amount * 1.04);
```

Comment:
```suggestion
		return round(amount * 1.04);
```

?

Reply:
`round` caused a circular dependency, found a similar function called `roundDp` (no idea what dp means) but really was just to get around that dependency issue. Will take another look

Reply:
dp => decimal point?

Reply:
🤦  yep that occurred to and I renamed the function